### PR TITLE
ensure that the script works in Linux.

### PR DESCRIPTION
**Pull request by joejoseph00**
When I ran this script in linux, I had to make this change in order for it to work.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+RUN ln -s /usr/bin/perl /opt/common/perl/bin/perl
+RUN cpanm HTML::Form
+RUN cpanm IO::Socket::SSL
+RUN cpanm Term::ReadKey
+RUN cpanm LWP::Protocol::https
+
+RUN mkdir /opt/wpss/wpss_scan_results
+RUN chmod 777 /opt/wpss/wpss_scan_results -R
+
+RUN /opt/wpss/install.pl
```
Comment (morvans) [inline Dockerfile:30]: I think there's a missing `\` at the end of the line.

Comment (joejoseph00) [reply Dockerfile:30]: Yes I think you're correct, I'll make a change, sec

Comment (joejoseph00) [reply Dockerfile:30]: PR is conflicting now, you can fork my repo if you want.
or try to resolve the upstream conflicts
https://github.com/wet-boew/wet-boew-wpss/pull/122

Comment (morvans) [reply Dockerfile:30]: I'll try that but it seems that there is new linux-incompatible changes in `master`.  :(

**Pull request closed without merging**